### PR TITLE
Fix test disabling

### PR DESCRIPTION
**Pull request by bader**
"UNSUPPORTED: TEMPORARY_DISABLED" doesn't really disable the test.
"REQUIRES: TEMPORARY_DISABLED" disables because test requires feature,
which is not enabled.

### CODE REVIEW

**Review by v-klochkov — DISMISSED** [review]

File: SYCL/ESIMD/fp_call_from_func.cpp [hunk 1]
```diff
@@ -7,7 +7,7 @@
 //===----------------------------------------------------------------------===//
 // REQUIRES: gpu
 // Issue #162 Test timeouts on Windows and Linux
```
Comment (bader) [inline SYCL/ESIMD/fp_call_from_func.cpp]:
```suggestion
// Issue #163 Test timeouts on Windows and Linux
```
@vladimirlaz, it looks like the right issue number is 163, not 162. Am I right?

File: SYCL/Basic/get_backend.cpp [hunk 2]
```diff
@@ -1,4 +1,4 @@
-// UNSUPPORTED: TEMPORARY_DISABLED
+// REQUIRES: TEMPORARY_DISABLED
 // RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
 // RUN: env SYCL_DEVICE_FILTER=%sycl_be %t.out
```
Comment (bader) [inline SYCL/Basic/get_backend.cpp]: @vladimirlaz, either the comment below or this RUN command must be updated - they contradict each other.

// Do not set SYCL_DEVICE_FILTER. We do not want the preferred
// backend.

File: SYCL/ESIMD/fp_call_from_func.cpp (hunk 1 above)
Comment (vladimirlaz) [reply SYCL/ESIMD/fp_call_from_func.cpp]: you are right

File: SYCL/Basic/get_backend.cpp (hunk 2 above)
Comment (vladimirlaz) [inline SYCL/Basic/get_backend.cpp]: According to test logic it should be
```suggestion
// RUN: %t.out
```

**Pull request merged**